io.embrace.sdk/Scripts/EmbraceStartupArgs.cs

```diff
@@ -0,0 +1,20 @@
+namespace EmbraceSDK.Internal
+{
+    public struct EmbraceStartupArgs
```

Comment:
This symbol is referenced from the public `EmbraceSDK` namespace in `Embrace.cs` which I assume makes it publicly accessible. I'd suggest renaming the namespace to match, and adding public facing docs

Reply:
Thank you! I'll rename the namespace and add the docs requested.